brevbaker/src/main/kotlin/no/nav/brev/brevbaker/template/Letter.kt

```diff
@@ -5,12 +5,28 @@ import no.nav.pensjon.brevbaker.api.model.Felles
 // TODO: Look at * projections for LetterTemplate, we have to have it for the API endpoint, but perhaps not for internal usage.
 // TODO: There is a bug where if you have a template with say EmptyBrevData as upper bound for ParameterType,
 //        when passing it to Letter you can pass any value as argument, and a new upper bound of Any will be chosen.
-data class Letter<ParameterType : Any>(val template: LetterTemplate<*, ParameterType>, val argument: ParameterType, val language: Language, val felles: Felles) {
+
+@InterneDataklasser
+data class LetterImpl<ParameterType : Any>(
```

Comment:
Jeg har alltid fått vondt av `Impl` på klasser og det motsatte `I` på interfaces. Ikke at jeg har noe godt forslag i dette tilfellet da.

Reply:
Ja, eg liker det heller ikkje. I valet mellom dei to liker eg `Impl`-varianten best, men det er mest i mangel på noko betre. `LetterInternalImpl`? `InternalLetter` kling litt feil.

Reply:
Helt enig. Jeg ville bare få det sagt. Siden interfacene er det vi eksponerer i API'et vårt så ønsker vi ikke at de skal ha merkelige navn.

Reply:
Det er jo mer eller mindre en DTO dette da. Så det kan være en mulighet, ikke at det er så veldig forskjellig fra Impl.